release-notes/2025-08/README.md

```diff
@@ -0,0 +1,179 @@
+# SystemLink Enterprise 2025-08 Release Notes
+
+The 2025-08 release for SystemLink Enterprise has been published to
+<https://downloads.artifacts.ni.com>. This update includes new features, bug
+fixes, and security updates. Work with your account representative to obtain
+credentials to access these artifacts. If you are not upgrading from the
+previous release, refer to past release notes to ensure you have addressed all
+required configuration changes.
+
+## New Features and Behavior changes
+
+- Support for Server-Side Encryption through the Key Management Service (AWS
+  SSE-KMS). For more information, refer to
+  [Configuring File Storage](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/configuring-file-storage.html).
+- `dataframeservice:1.20.72`
+  - The DataFrame Service now limits the amount of property metadata that tables
+    and columns can contain. This limit is configurable from the Helm values via
+    `dataframeservice.ingestion.maxPropertyMetadataLength`.
+  - This limit sets the maximum total character count for all property metadata
```

Comment:
I don't think we need to include 'columns' in this text as that is part of the table. Suggestion:

'This limit sets the maximum total character count for all property metadata keys and values across the table.'